fix(curriculum): Provide Example for Media Query

hanswang123456 opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #47417

<!-- Feel free to add any additional description of changes below this line -->

hanswang123456 committed:
Update 612ec19d5268da7074941f84.md

hanswang123456 committed:
Switched order

Sboonny commented on curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md:
```diff
@@ -6,6 +6,14 @@ dashedName: step-32
 # --description--
+In order to query 2 media condition, the `and` keyword is used.
+
+For example, a media query that targets a display width between 500px and 1000px would be:
```
to fix the eslinit issue, you can add space between the quotes and this line

```suggestion
For example, a media query that targets a display width between 500px and 1000px would be:

```

Sboonny reviewed (commented):
I think eslint issue because of the lack of empty lines.

edit: Well, didn't mean to comment twice sorry about that

Tracerract commented:
Thank you for drafting this! I really like the simple explanation, though I feel like the other logical operators are excluded with this particular wording. Even if they aren't explained, I still feel they're worth mentioning here. Maybe something like:

> Certain at-rules can query multiple conditions with the use of logical operators like `and`, `or`, as well as `not`.

Also the requirement explanation might be better with 'wider' or 'taller' instead of 'bigger', though that's just semantics.

hanswang123456 committed:
Update curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md

Co-authored-by: Muhammed Mustafa <muhammed@freecodecamp.org>

hanswang123456 commented:
@Tracerract I found some links on the topics. https://www.freecodecamp.org/news/media-query-css-example-max-and-min-screen-width-for-mobile-responsive-design/. Maybe we can put in the link as further reading?

hanswang123456 committed:
Update 612ec19d5268da7074941f84.md

hanswang123456 commented:
I added the link to the FreeCodeCamp article as reference for other queries and changed the larger to "wider" but havent changed the smaller yet(narrower could be an option).

Tracerract commented:
I definitely like that link, all the forum posts I found when I needed help were recommending external references, so having a FCC article to reference questions to is nice.

Since you did change the first term to specify width (wider), width is implied for the second term (smaller), where before it wasn't directly implied like that (bigger, smaller). I like the way you have it now, and just all the changes so far really.  The only thing I see is a typo here:

> In order to query 2 media condition,
Where it should be plural, `conditions`.

hanswang123456 committed:
Update 612ec19d5268da7074941f84.md

lasjorg commented on curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md:
```diff
@@ -6,8 +6,17 @@ dashedName: step-32
 ---
 
 # --description--
+In order to query two media conditions, the `and` logical operator is used. This one of many [logical operators](https://www.freecodecamp.org/news/media-query-css-example-max-and-min-screen-width-for-mobile-responsive-design/) used to construct media queries.
```
The challenge should not contain outbound links.

> [Challenge descriptions/instructions](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions)
> 
> Don't use outbound links. These interrupt the flow. Campers should never have to google anything during these challenges.

hanswang123456 replied:
Ill remove the link and convert to PR.

hanswang123456 committed:
Update 612ec19d5268da7074941f84.md

jeremylt commented on curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md:
```diff
@@ -6,8 +6,17 @@ dashedName: step-32
 ---
 
 # --description--
+In order to query two media conditions, the `and` logical operator is used. This one of many used to construct media queries.
```
I think this is a bit more straightforward language:
```suggestion
Logical operators can be used to construct more complex media queries. The `and` logical operator is used to query two media conditions.
```

Sboonny replied:
I have edited Jeremy comment for the needed format, and to avoid spamming the pr.

hanswang123456 committed:
Update curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ec19d5268da7074941f84.md

Co-authored-by: Jeremy L Thompson <jeremy@jeremylt.org>

hanswang123456 commented:
My bad for not responding sooner, I've just moved to university.

hanswang123456 commented:
I've compiled the suggestions and opened as a active PR

Sboonny reviewed (approved):
LGTM 🎉, thank you for your contribution to the page! 👍.

lasjorg reviewed (approved):
LGTM
